hlint/hlint.dhall

```diff
@@ -901,7 +901,7 @@ in [ Rule.Arguments { arguments =
    -- Enum
    , hintNote "[minBound .. maxBound]" "universe" "Use `universe` from `Relude.Extra.Enum`"
    , hintNote "succ" "next" "`succ` from `Prelude` is a pure function but it may throw exception. Consider using `next` from `Relude.Extra.Enum` instead."
-   , hintNote "pred" "prec" "`pred` from `Prelude` is a pure function but it may throw exception. Consider using `prec` from `Relude.Extra.Enum` instead."
+   , hintNote "pred" "prev" "`pred` from `Prelude` is a pure function but it may throw exception. Consider using `prev` from `Relude.Extra.Enum` instead."
```

Comment:
Could you, please, regenerate `.hlint.yaml` from this file? `.hlint.yaml` should be generated from the `hlint.dhall` file. Instead of generating (if generating is too much work) you can just update the text in `.hlint.yaml` since it's relatively easy in this particular case :slightly_smiling_face: 

After looking more closely at this rule and issue #181 I decided that removing these rules is not a good idea. It's better to change them, so I've updated issue #181 and removed it from `0.6.0.0` milestone.